msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java

```diff
@@ -49,6 +50,25 @@
 
     private static final String TAG = BrokerClientIdRefreshTokenAccessor.class.getSimpleName();
 
+    private static void throwIfNotValidBroker(final Context context) throws MsalClientException {
+        final BrokerValidator brokerValidator = new BrokerValidator(context);
+        final Set<BrokerData> validBrokers = brokerValidator.getValidBrokers();
+
+        for (final BrokerData brokerData : validBrokers) {
+            if (brokerData.packageName.equals(context.getPackageName())) {
```

Comment:
I'm thinking we want a getBrokerData(packageName) method here.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1274/commits/31f24b52a4d1e8dc035473a18c2c552b9e3b7b9d